.evergreen/abidiff-test.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+declare working_dir
+working_dir="$(pwd)"
+
+export PATH
+PATH="${working_dir:?}/install/bin:${PATH:-}"
+
+declare -a common_flags
+flags=(
+  --headers-dir1 install/old/include
+  --headers-dir2 install/new/include
+  --non-reachable-types
+  --fail-no-debug-info
+)
+
+declare -a abi_flags=("${common_flags[@]}" --suppressions cxx-abi/abignore)
+declare -a noabi_flags=("${common_flags[@]}" --suppressions cxx-noabi/abignore)
+
+command -V abidiff >/dev/null
+
+mkdir cxx-abi cxx-noabi
+
+cat >cxx-abi/abignore <<DOC
+[suppress_type]
+name_not_regexp = ^(bsoncxx|mongocxx)::v[[:digit:]]+::
+
+[suppress_function]
+name_not_regexp = ^(bsoncxx|mongocxx)::v[[:digit:]]+::
+
+[suppress_variable]
+name_not_regexp = ^(bsoncxx|mongocxx)::v[[:digit:]]+::
+DOC
+
+cat >cxx-noabi/abignore <<DOC
+[suppress_type]
+name_regexp = ^(bsoncxx|mongocxx)::v[[:digit:]]+::
+
+[suppress_function]
+name_regexp = ^(bsoncxx|mongocxx)::v[[:digit:]]+::
+
+[suppress_variable]
+name_regexp = ^(bsoncxx|mongocxx)::v[[:digit:]]+::
+DOC
+
+# Allow task to upload the diff reports despite failed status.
+echo "Comparing stable ABI for bsoncxx..."
+if ! abidiff "${abi_flags[@]}" install/old/lib/libbsoncxx.so install/new/lib/libbsoncxx.so &>cxx-abi/bsoncxx.txt; then
+  declare status
+  status='{"status":"failed", "type":"test", "should_continue":true, "desc":"abidiff returned an error for bsoncxx (stable)"}'
+  curl -sS -d "${status:?}" -H "Content-Type: application/json" -X POST localhost:2285/task_status || true
+  :
```

Comment:
Remove stray `:`? Same question applies below.

Reply:
Leftover from when porting over the HTTP endpoint commands from the abi-compliance-check scripts. Removed as suggested.